middle_end/flambda/simplify/env/downwards_env.mli

```diff
@@ -215,3 +215,21 @@ val add_use_of_closure_var : t -> Var_within_closure.t -> t
 val closure_var_uses : t -> Var_within_closure.Set.t
 
 val without_closure_var_uses : t -> t
+
+type closure_info =
```

Comment:
This can be `private`, no?

Reply:
I split the type into its own module, and made the type private (given its current simplicity, it's not really that useful, but if we extend it in the future, it might help things)